omnia/bin/oracle-transporter-ssb

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+_here=$(cd "${BASH_SOURCE[0]%/*}" && pwd)
+_command="$1"
+
+. "$_here/../lib/source.sh"
+
+OMNIA_MSG_LIMIT=20
+
+#TODO: maybe use `exec`, but exported unctions won't work probably
+case "$_command" in
+	publish) ssb-server publish . <<< "$2";;
```

Comment:
Since we now know, that the message size will be too large when we use gofer's trace I would make the message be sent to `oracle-transporter-ssb` through `stdin`

Reply:
I don't know, when I run `getconf ARG_MAX` on my Arch Linux machine I get about 2Mb that I can put on my command line. Lowest number I've seen reported is around 100kb so still well within our design limits. It could of course be configured lower.

But if we are to change the way to pass arguments to CLI in the MOP we should first consider how the interfaces translate to passing through STDIN.

Reply:
I just came up with the idea that args will be used for ad-hoc config params and STDIN as the payload. 
Would that be a good way of looking at things?